packages/terra-date-picker/src/DatePicker.module.scss

```diff
@@ -1,90 +1,185 @@
-/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+/* stylelint-disable max-nesting-depth */
 // Themes
 @import './clinical-lowlight-theme/DatePicker.module';
 @import './orion-fusion-theme/DatePicker.module';
 
 :local {
   .date-picker {
     display: inline-block;
+  }
+
+  .date-input-container {
+    display: inline-flex;
+  }
+
+  .date-input {
+    background-color: var(--terra-date-picker-input-background-color, #fff);
+    border-bottom-right-radius: 0;
+    border-color: var(--terra-date-picker-input-border-color, #dedfe0);
+    border-radius: var(--terra-date-picker-input-border-radius, 3px);
+    border-style: var(--terra-date-picker-input-border-style, solid);
+    border-top-right-radius: 0;
+    border-width: var(--terra-date-picker-input-border-width, 1px);
+    box-sizing: content-box;
+    color: var(--terra-date-picker-input-color, #1c1f21);
+    display: inline-block;
+    padding-bottom: var(--terra-date-picker-input-padding-bottom, 0.2857142858rem);
+    padding-left: var(--terra-date-picker-input-padding-left, 0.2857142858rem);
+    padding-right: var(--terra-date-picker-input-padding-right, 0.2857142858rem);
+    padding-top: var(--terra-date-picker-input-padding-top, 0.2857142858rem);
```

Comment:
to reduce the number of theme vars, could this be simplified to use shorthand sytax?

Reply:
I had referenced `time-input` for this change, but it does seem appropriate to just have this consolidated as `padding`.
Changed here : https://github.com/cerner/terra-framework/pull/1208/commits/ba9d3c9b977ded21c1db0789cbe2b1e43dd3c7c6